odl/space/fspace.py

```diff
@@ -216,7 +216,7 @@ class FunctionSetVector(Operator):
 
     """Representation of a `FunctionSet` element."""
 
-    def __init__(self, fset, fcall, out_dtype=None):
```

Comment:
It's not used anywhere, and it only makes sense as a parameter for the space (this is the vector class). Also, the almost-deleted documentation was another hint :-)